lib/headers.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+import re
+
+from m2ee import logger  # noqa: E402
```

Comment:
We don't actually need this `noqa: E402`, (E402 module level import not at top of file), as the module level imports _are_ at the top of the file.